internal/db/fetcher/versioned.go

```diff
@@ -381,7 +386,7 @@ func (vf *VersionedFetcher) merge(c cid.Cid) error {
 			keys.DataStoreKey{
 				CollectionShortID: shortID,
 				DocID:             string(block.Delta.GetDocID()),
-				FieldID:           fmt.Sprint(field.ID),
+				FieldID:           fmt.Sprint(fieldShortID),
```

Comment:
todo: Please use `strconv.Itoa` instead of `fmt.Sprint`. It has 0 allocations instead of 2 and is about an order of magnitude faster.

Reply:
This is not new, but I might have a look in this PR.

Reply:
My attention needs to be on Lens in order to unblock Shinzo so I'm going to leave this as-is.